catena4801_simple/README.md

```diff
@@ -0,0 +1,312 @@
+remote: Total 1201 (delta 27), reused 24 (delta 14), pack-reused 1151
+Receiving objects: 100% (1201/1201), 275.99 KiB | 0 bytes/s, done.
+Resolving deltas: 100% (900/900), done.
+...
+
+==== Summary =====
+No repos with errors
+No repos skipped.
+*** no repos were pulled ***
+Repos downloaded:      Catena-Arduino-Platform arduino-lorawan Catena-mcciadk arduino-lmic Adafruit_FRAM_I2C Modbus-for-Arduino
+```
+
+It has a number of advanced options; use `../git-boot.sh -h` to get help, or look at the source code [here](https://github.com/mcci-catena/Catena-Sketches/blob/master/git-boot.sh).
+
+**Beware of issue #18**.  If you happen to already have libraries installed with the same names as any of the libraries in `git-repos.dat`, `git-boot.sh` will silently use the versions of the library that you already have installed. (We hope to soon fix this to at least tell you that you have a problem.)
+
+#### List of required libraries
+
+This sketch depends on the following libraries.
+
+*  https://github.com/mcci-catena/Catena-Arduino-Platform
+*  https://github.com/mcci-catena/arduino-lorawan
+*  https://github.com/mcci-catena/Catena-mcciadk
+*  https://github.com/mcci-catena/arduino-lmic
+*  https://github.com/mcci-catena/Adafruit_FRAM_I2C
+*  https://github.com/mcci-catena/Modbus-for-Arduino
+
+### Build and Download
+
+Shutdown the Arduino IDE and restart it, just in case.
+
+Ensure selected board is 'MCCI Catena 4801' (in the GUI, check that `Tools`>`Board "..."` says `"MCCI Catena 4801"`.
+
+In the IDE, use File>Open to load the `catena4801_simple.ino` sketch. (Remember, in step 1 you cloned `Catena-Sketches` -- find that, and navigate to `{somewhere}/Catena-Sketches/catena4801_simple/`)
+
+Follow normal Arduino IDE procedures to build the sketch: `Sketch`>`Verify/Compile`. If there are no errors, go to the next step.
+
+### Configure Modbus parameters
+
+To communicate with the Modbus slave, it is necessary to modify the Modbus slave parameter in the `setup_modbus` function as it matches the device parameters.
+
+```
+/* Modbus slave parameters */
+u8addr = 65;			// current target device: 01
+u8fct = 03;			// Function code
+u16RegAdd = 00;			// start address in device
+u16CoilsNo = 13;		// number of elements (coils or registers) to read
+host.begin(&mySerial, 9600);	// baud-rate at 9600
+```
+
+Connect your modbus slave�s A and B terminal to Catena 4801�s A and B terminal (JP3).
+
+### Load the sketch into the Catena
+
+Make sure the correct port is selected in `Tools`>`Port`.
+
+Load the sketch into the Catena using `Sketch`>`Upload` and move on to provisioning.
+
+## Set the identity of your Catena 4801
+
+This can be done with any terminal emulator, but it's easiest to do it with the serial monitor built into the Arduino IDE or with the equivalent monitor that's part of the Visual Micro IDE. It can also be done usign Tera Term.
+
+### Check platform amd serial number setup
+
+![Newline](./assets/serial-monitor-newline.png)
```

Comment:
It looks link is wrong or file is missing.

Reply:
Added the file to the directory